src/fields.ts

```diff
@@ -27,7 +28,7 @@ export const pbSchemaTypescriptMap = {
 
   // Dependent on schema
   file: (fieldSchema: FieldSchema) =>
-    fieldSchema.maxSelect && fieldSchema.maxSelect > 1 ? "string[]" : "string",
+    fieldSchema.maxSelect && fieldSchema.maxSelect > 1 ? "string[] | File[]" : "string | File",
```

Comment:
I think this change should be reverted because it is misleading. The API is always going to store and return a string.

Instead, you could add a helper that adds File types for create/update